script/fork-helpers/SimulateVote.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-FileCopyrightText: 2023 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.24;
+
+import "forge-std/Script.sol";
+import "forge-std/console.sol";
+import { DeploymentFixtures } from "test/helpers/Fixtures.sol";
+import { IStakingRouter } from "../../src/interfaces/IStakingRouter.sol";
+import { IBurner } from "../../src/interfaces/IBurner.sol";
+
+contract SimulateVote is Script, DeploymentFixtures {
+    function run() external {
+        Env memory env = envVars();
+        initializeFromDeployment(env.DEPLOY_CONFIG);
+
+        IStakingRouter stakingRouter = IStakingRouter(locator.stakingRouter());
+        IBurner burner = IBurner(locator.burner());
+
+        address agent = stakingRouter.getRoleMember(
+            stakingRouter.STAKING_MODULE_MANAGE_ROLE(),
+            0
+        );
+        vm.label(agent, "agent");
+        console.logUint(agent.balance);
```

Comment:
I assume we need for a check, and not just a logging.

Reply:
this is a debug thing actually, thx